sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1057,100 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of multinomial function.
```

Comment:
Please don't reproduce wikipedia entries here. Simply state what it does and refer the reader to the [article](https://en.wikipedia.org/wiki/Multinomial_theorem).